packages/fxa-settings/src/lib/gql-key-stretch-upgrade.ts

```diff
@@ -67,7 +67,10 @@ export async function tryFinalizeUpgrade(
   } finally {
     // Clear out the state. No reason to keep trying this...
     // Note that the upgradeClient will report issues to Sentry.
-    sensitiveDataClient.KeyStretchUpgradeData = undefined;
+    sensitiveDataClient.setDataType(
```

Comment:
Eek, missed this one in the previous large patch that landed yesterday after fixing the conflicts during the rebase.

Reply:
No longer relevant.